tests/tests_integration/test_session/test_graph_flow/test_uplift_workflow_to_python.yml

```diff
@@ -3936,19 +3978,28 @@ rules:
     neatId: http://purl.org/cognite/neat/data-model/verified/physical/sp_windfarm/WindFarm/v1/Cognite360ImageAnnotation
     view: cdf_cdm:Cognite360ImageAnnotation(version=v1)
   - description: Represents a logical collection of Cognite360Image instances
+    filter_: 'rawFilter({"and": [{"hasData": [{"type": "container", "space": "cdf_cdm_3d",
+      "externalId": "Cognite3DRevision"}]}, {"equals": {"property": ["cdf_cdm_3d",
+      "Cognite3DModel", "type"], "value": "Image360"}}]})'
     implements: cdf_cdm:CogniteDescribable(version=v1),cdf_cdm:Cognite3DRevision(version=v1)
     in_model: true
     logical: http://purl.org/cognite/neat/data-model/verified/logical/sp_windfarm/WindFarm/v1/Cognite360ImageCollection
     neatId: http://purl.org/cognite/neat/data-model/verified/physical/sp_windfarm/WindFarm/v1/Cognite360ImageCollection
     view: cdf_cdm:Cognite360ImageCollection(version=v1)
   - description: Navigational aid for traversing Cognite360ImageModel instances
+    filter_: 'rawFilter({"and": [{"hasData": [{"type": "container", "space": "cdf_cdm_3d",
+      "externalId": "Cognite3DModel"}]}, {"equals": {"property": ["cdf_cdm_3d", "Cognite3DModel",
+      "type"], "value": "Image360"}}]})'
```

Comment:
This test has not been sunning for a while, and this is the result of an already merged change.